gradle/libs.versions.toml

```diff
@@ -0,0 +1,96 @@
+## Generated by $ ./gradlew refreshVersionsCatalog
+
+[bundles]
+
+
+
+[plugins]
+
+detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
+
+[versions]
+
+kotlin = "1.9.20"
+
+android-gradle-plugin = "8.2.0-rc03"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/ca75b5d9b54bad1f080d3a092600b3a912a91155/gradle/libs.versions.toml#L15"></span>A newer version of com.android.tools.build:gradle than 8.2.0-rc03 is available: 8.2.1</td>
    </tr>
  </tbody>
</table>